endpoints/api/billing.py

```diff
@@ -1019,7 +1058,7 @@ def delete(self, orgname, subscription_id):
                 return ("Organization not valid", 400)
 
             model.organization_skus.remove_subscription_from_org(organization.id, subscription_id)
-            return ("Deleted", 204)
+            return ("Deleted", 200)
```

Comment:
Any reason to change from 204 to 200?

Reply:
I was having trouble updating the frontend component when removing a sku because 204 returned an empty response